sha1-name.c

```diff
@@ -1773,12 +1773,12 @@ static int get_oid_with_context_1(const char *name,
 			if (new_filename)
```

Comment:
commit subject: `make ... more gentle` (it's an adjective in this form, not an adverb).